Bootstraps/AsyncInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace PHPPM\Bootstraps;
+
+
+use React\EventLoop\LoopInterface;
+
+interface AsyncInterface
```

Comment:
That interface is redundant since you pass the loop already in the BridgeInterface `bootstrap` method.